package/yast2-packager.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Tue Aug 23 11:17:36 UTC 2016 - jreidinger@suse.com
+
+- always enable next in license confirmation dialog, because even
```

Comment:
- NP:  *always enable next* might be confusing, I'd use *always enable "Next" button*
- NP2: Start with capital letter to be unified with the rest